data/trie/patriciaMerkleTrie.go

```diff
@@ -72,7 +72,12 @@ func (tr *patriciaMerkleTrie) Get(key []byte) ([]byte, error) {
 	}
 	hexKey := keyBytesToHex(key)
 
-	return tr.root.tryGet(hexKey, tr.trieStorage.Database())
+	val, err := tr.root.tryGet(hexKey, tr.trieStorage.Database())
+	if err != nil {
```

Comment:
We should not follow the pattern in which you both log and return the error as you might get double prints
Maybe switch this to Trace?

Reply:
Done.